`beets` does not build due to pytest build failure

## Issue description

Derivation `81kl20ksz7wjv6anvkmw2bxi6ki5y6z3-python3.7-pytest-3.10.1.drv` fails to build with the following log:

<details>
<summary>nix log 81kl20ksz7wjv6anvkmw2bxi6ki5y6z3-python3.7-pytest-3.10.1.drv</summary>

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/ai5x3bm799035kaj5abxrqickyfrgrmc-pytest-3.10.1.tar.gz
source root is pytest-3.10.1
setting SOURCE_DATE_EPOCH to timestamp 1541957655 of file pytest-3.10.1/src/pytest.egg-info/SOURCES.txt
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
@nix { "action": "setPhase", "phase": "buildPhase" }
building
running bdist_wheel
running build
running build_py
creating build
creating build/lib
copying src/pytest.py -> build/lib
creating build/lib/_pytest
copying src/_pytest/warnings.py -> build/lib/_pytest
copying src/_pytest/warning_types.py -> build/lib/_pytest
copying src/_pytest/unittest.py -> build/lib/_pytest
copying src/_pytest/tmpdir.py -> build/lib/_pytest
copying src/_pytest/terminal.py -> build/lib/_pytest
copying src/_pytest/stepwise.py -> build/lib/_pytest
copying src/_pytest/skipping.py -> build/lib/_pytest
copying src/_pytest/setupplan.py -> build/lib/_pytest
copying src/_pytest/setuponly.py -> build/lib/_pytest
copying src/_pytest/runner.py -> build/lib/_pytest
copying src/_pytest/resultlog.py -> build/lib/_pytest
copying src/_pytest/reports.py -> build/lib/_pytest
copying src/_pytest/recwarn.py -> build/lib/_pytest
copying src/_pytest/python_api.py -> build/lib/_pytest
copying src/_pytest/python.py -> build/lib/_pytest
copying src/_pytest/pytester.py -> build/lib/_pytest
copying src/_pytest/pathlib.py -> build/lib/_pytest
copying src/_pytest/pastebin.py -> build/lib/_pytest
copying src/_pytest/outcomes.py -> build/lib/_pytest
copying src/_pytest/nose.py -> build/lib/_pytest
copying src/_pytest/nodes.py -> build/lib/_pytest
copying src/_pytest/monkeypatch.py -> build/lib/_pytest
copying src/_pytest/main.py -> build/lib/_pytest
copying src/_pytest/logging.py -> build/lib/_pytest
copying src/_pytest/junitxml.py -> build/lib/_pytest
copying src/_pytest/hookspec.py -> build/lib/_pytest
copying src/_pytest/helpconfig.py -> build/lib/_pytest
copying src/_pytest/freeze_support.py -> build/lib/_pytest
copying src/_pytest/fixtures.py -> build/lib/_pytest
copying src/_pytest/doctest.py -> build/lib/_pytest
copying src/_pytest/deprecated.py -> build/lib/_pytest
copying src/_pytest/debugging.py -> build/lib/_pytest
copying src/_pytest/compat.py -> build/lib/_pytest
copying src/_pytest/capture.py -> build/lib/_pytest
copying src/_pytest/cacheprovider.py -> build/lib/_pytest
copying src/_pytest/_version.py -> build/lib/_pytest
copying src/_pytest/_argcomplete.py -> build/lib/_pytest
copying src/_pytest/__init__.py -> build/lib/_pytest
creating build/lib/_pytest/assertion
copying src/_pytest/assertion/util.py -> build/lib/_pytest/assertion
copying src/_pytest/assertion/truncate.py -> build/lib/_pytest/assertion
copying src/_pytest/assertion/rewrite.py -> build/lib/_pytest/assertion
copying src/_pytest/assertion/__init__.py -> build/lib/_pytest/assertion
creating build/lib/_pytest/_code
copying src/_pytest/_code/source.py -> build/lib/_pytest/_code
copying src/_pytest/_code/code.py -> build/lib/_pytest/_code
copying src/_pytest/_code/_py2traceback.py -> build/lib/_pytest/_code
copying src/_pytest/_code/__init__.py -> build/lib/_pytest/_code
creating build/lib/_pytest/mark
copying src/_pytest/mark/structures.py -> build/lib/_pytest/mark
copying src/_pytest/mark/legacy.py -> build/lib/_pytest/mark
copying src/_pytest/mark/evaluate.py -> build/lib/_pytest/mark
copying src/_pytest/mark/__init__.py -> build/lib/_pytest/mark
creating build/lib/_pytest/config
copying src/_pytest/config/findpaths.py -> build/lib/_pytest/config
copying src/_pytest/config/exceptions.py -> build/lib/_pytest/config
copying src/_pytest/config/argparsing.py -> build/lib/_pytest/config
copying src/_pytest/config/__init__.py -> build/lib/_pytest/config
installing to build/bdist.linux-x86_64/wheel
running install
running install_lib
creating build/bdist.linux-x86_64
creating build/bdist.linux-x86_64/wheel
creating build/bdist.linux-x86_64/wheel/_pytest
creating build/bdist.linux-x86_64/wheel/_pytest/config
copying build/lib/_pytest/config/__init__.py -> build/bdist.linux-x86_64/wheel/_pytest/config
copying build/lib/_pytest/config/argparsing.py -> build/bdist.linux-x86_64/wheel/_pytest/config
copying build/lib/_pytest/config/exceptions.py -> build/bdist.linux-x86_64/wheel/_pytest/config
copying build/lib/_pytest/config/findpaths.py -> build/bdist.linux-x86_64/wheel/_pytest/config
creating build/bdist.linux-x86_64/wheel/_pytest/mark
copying build/lib/_pytest/mark/__init__.py -> build/bdist.linux-x86_64/wheel/_pytest/mark
copying build/lib/_pytest/mark/evaluate.py -> build/bdist.linux-x86_64/wheel/_pytest/mark
copying build/lib/_pytest/mark/legacy.py -> build/bdist.linux-x86_64/wheel/_pytest/mark
copying build/lib/_pytest/mark/structures.py -> build/bdist.linux-x86_64/wheel/_pytest/mark
creating build/bdist.linux-x86_64/wheel/_pytest/_code
copying build/lib/_pytest/_code/__init__.py -> build/bdist.linux-x86_64/wheel/_pytest/_code
copying build/lib/_pytest/_code/_py2traceback.py -> build/bdist.linux-x86_64/wheel/_pytest/_code
copying build/lib/_pytest/_code/code.py -> build/bdist.linux-x86_64/wheel/_pytest/_code
copying build/lib/_pytest/_code/source.py -> build/bdist.linux-x86_64/wheel/_pytest/_code
creating build/bdist.linux-x86_64/wheel/_pytest/assertion
copying build/lib/_pytest/assertion/__init__.py -> build/bdist.linux-x86_64/wheel/_pytest/assertion
copying build/lib/_pytest/assertion/rewrite.py -> build/bdist.linux-x86_64/wheel/_pytest/assertion
copying build/lib/_pytest/assertion/truncate.py -> build/bdist.linux-x86_64/wheel/_pytest/assertion
copying build/lib/_pytest/assertion/util.py -> build/bdist.linux-x86_64/wheel/_pytest/assertion
copying build/lib/_pytest/__init__.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/_argcomplete.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/_version.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/cacheprovider.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/capture.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/compat.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/debugging.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/deprecated.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/doctest.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/fixtures.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/freeze_support.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/helpconfig.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/hookspec.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/junitxml.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/logging.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/main.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/monkeypatch.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/nodes.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/nose.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/outcomes.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/pastebin.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/pathlib.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/pytester.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/python.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/python_api.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/recwarn.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/reports.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/resultlog.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/runner.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/setuponly.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/setupplan.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/skipping.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/stepwise.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/terminal.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/tmpdir.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/unittest.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/warning_types.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/_pytest/warnings.py -> build/bdist.linux-x86_64/wheel/_pytest
copying build/lib/pytest.py -> build/bdist.linux-x86_64/wheel
running install_egg_info
running egg_info
writing src/pytest.egg-info/PKG-INFO
writing dependency_links to src/pytest.egg-info/dependency_links.txt
writing entry points to src/pytest.egg-info/entry_points.txt
writing requirements to src/pytest.egg-info/requires.txt
writing top-level names to src/pytest.egg-info/top_level.txt
reading manifest file 'src/pytest.egg-info/SOURCES.txt'
writing manifest file 'src/pytest.egg-info/SOURCES.txt'
Copying src/pytest.egg-info to build/bdist.linux-x86_64/wheel/pytest-3.10.1-py3.7.egg-info
running install_scripts
creating build/bdist.linux-x86_64/wheel/pytest-3.10.1.dist-info/WHEEL
creating 'dist/pytest-3.10.1-py2.py3-none-any.whl' and adding 'build/bdist.linux-x86_64/wheel' to it
adding 'pytest.py'
adding '_pytest/__init__.py'
adding '_pytest/_argcomplete.py'
adding '_pytest/_version.py'
adding '_pytest/cacheprovider.py'
adding '_pytest/capture.py'
adding '_pytest/compat.py'
adding '_pytest/debugging.py'
adding '_pytest/deprecated.py'
adding '_pytest/doctest.py'
adding '_pytest/fixtures.py'
adding '_pytest/freeze_support.py'
adding '_pytest/helpconfig.py'
adding '_pytest/hookspec.py'
adding '_pytest/junitxml.py'
adding '_pytest/logging.py'
adding '_pytest/main.py'
adding '_pytest/monkeypatch.py'
adding '_pytest/nodes.py'
adding '_pytest/nose.py'
adding '_pytest/outcomes.py'
adding '_pytest/pastebin.py'
adding '_pytest/pathlib.py'
adding '_pytest/pytester.py'
adding '_pytest/python.py'
adding '_pytest/python_api.py'
adding '_pytest/recwarn.py'
adding '_pytest/reports.py'
adding '_pytest/resultlog.py'
adding '_pytest/runner.py'
adding '_pytest/setuponly.py'
adding '_pytest/setupplan.py'
adding '_pytest/skipping.py'
adding '_pytest/stepwise.py'
adding '_pytest/terminal.py'
adding '_pytest/tmpdir.py'
adding '_pytest/unittest.py'
adding '_pytest/warning_types.py'
adding '_pytest/warnings.py'
adding '_pytest/_code/__init__.py'
adding '_pytest/_code/_py2traceback.py'
adding '_pytest/_code/code.py'
adding '_pytest/_code/source.py'
adding '_pytest/assertion/__init__.py'
adding '_pytest/assertion/rewrite.py'
adding '_pytest/assertion/truncate.py'
adding '_pytest/assertion/util.py'
adding '_pytest/config/__init__.py'
adding '_pytest/config/argparsing.py'
adding '_pytest/config/exceptions.py'
adding '_pytest/config/findpaths.py'
adding '_pytest/mark/__init__.py'
adding '_pytest/mark/evaluate.py'
adding '_pytest/mark/legacy.py'
adding '_pytest/mark/structures.py'
adding 'pytest-3.10.1.dist-info/LICENSE'
adding 'pytest-3.10.1.dist-info/METADATA'
adding 'pytest-3.10.1.dist-info/WHEEL'
adding 'pytest-3.10.1.dist-info/entry_points.txt'
adding 'pytest-3.10.1.dist-info/top_level.txt'
adding 'pytest-3.10.1.dist-info/RECORD'
removing build/bdist.linux-x86_64/wheel
@nix { "action": "setPhase", "phase": "installPhase" }
installing
/build/pytest-3.10.1/dist /build/pytest-3.10.1
Processing ./pytest-3.10.1-py2.py3-none-any.whl
Requirement already satisfied: more-itertools>=4.0.0 in /nix/store/yy87byfhinkrwqkg7jrw6gg5pmv72jwv-python3.7-more-itertools-6.0.0/lib/python3.7/site-packages (from pytest==3.10.1) (6.0.0)
Requirement already satisfied: py>=1.5.0 in /nix/store/rckdy90j75r5g8f4444003madmgk4ka3-python3.7-py-1.7.0/lib/python3.7/site-packages (from pytest==3.10.1) (1.7.0)
Requirement already satisfied: attrs>=17.4.0 in /nix/store/nmzcj7vdqfch4jl4ndmkf1mvdyda4xs4-python3.7-attrs-18.2.0/lib/python3.7/site-packages (from pytest==3.10.1) (18.2.0)
Requirement already satisfied: six>=1.10.0 in /nix/store/imf48xfhb9dg8453gx131icwxrgwnwdv-python3.7-six-1.12.0/lib/python3.7/site-packages (from pytest==3.10.1) (1.12.0)
Requirement already satisfied: setuptools in /nix/store/pwqjs8nz67h4rid05vwma9z8fbsfmm16-python3.7-bootstrapped-pip-19.1.1/lib/python3.7/site-packages (from pytest==3.10.1) (41.0.1)
Requirement already satisfied: pluggy>=0.7 in /nix/store/ma1mczcg2x639dqiax99bda1g83nvw04-python3.7-pluggy-0.12.0/lib/python3.7/site-packages (from pytest==3.10.1) (0.12.0)
Requirement already satisfied: atomicwrites>=1.0 in /nix/store/dv9yfmqpq49qxk8bd435x61rflbfiwnb-python3.7-atomicwrites-1.3.0/lib/python3.7/site-packages (from pytest==3.10.1) (1.3.0)
Requirement already satisfied: importlib-metadata>=0.12 in /nix/store/s3cb27bv8m2g8izzs1pzvqvciyfis65v-python3.7-importlib-metadata-0.18/lib/python3.7/site-packages (from pluggy>=0.7->pytest==3.10.1) (0.18)
Requirement already satisfied: zipp>=0.5 in /nix/store/jyyvg50qmi0apdlcyaz7j3cfdkp15fc6-python3.7-zipp-0.5.1/lib/python3.7/site-packages (from importlib-metadata>=0.12->pluggy>=0.7->pytest==3.10.1) (0.5.1)
Installing collected packages: pytest
  WARNING: The scripts py.test and pytest are installed in '/nix/store/24c9c85nyzhvv129wdbn2g95hhax15l9-python3.7-pytest-3.10.1/bin' which is not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
Successfully installed pytest-3.10.1
/build/pytest-3.10.1
@nix { "action": "setPhase", "phase": "fixupPhase" }
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/24c9c85nyzhvv129wdbn2g95hhax15l9-python3.7-pytest-3.10.1
strip is /nix/store/qw8rhqk004qmqgnkvkpj85l1s194ang1-binutils-2.31.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/24c9c85nyzhvv129wdbn2g95hhax15l9-python3.7-pytest-3.10.1/lib  /nix/store/24c9c85nyzhvv129wdbn2g95hhax15l9-python3.7-pytest-3.10.1/bin 
patching script interpreter paths in /nix/store/24c9c85nyzhvv129wdbn2g95hhax15l9-python3.7-pytest-3.10.1
checking for references to /build/ in /nix/store/24c9c85nyzhvv129wdbn2g95hhax15l9-python3.7-pytest-3.10.1...
Rewriting #!/nix/store/0f331h72l3v42szvcqyc9pqg79ghid9m-python3-3.7.3/bin/python3.7 to #!/nix/store/0f331h72l3v42szvcqyc9pqg79ghid9m-python3-3.7.3
wrapping `/nix/store/24c9c85nyzhvv129wdbn2g95hhax15l9-python3.7-pytest-3.10.1/bin/pytest'...
Rewriting #!/nix/store/0f331h72l3v42szvcqyc9pqg79ghid9m-python3-3.7.3/bin/python3.7 to #!/nix/store/0f331h72l3v42szvcqyc9pqg79ghid9m-python3-3.7.3
wrapping `/nix/store/24c9c85nyzhvv129wdbn2g95hhax15l9-python3.7-pytest-3.10.1/bin/py.test'...
@nix { "action": "setPhase", "phase": "installCheckPhase" }
running install tests
============================= test session starts ==============================
platform linux -- Python 3.7.3, pytest-3.10.1, py-1.7.0, pluggy-0.12.0
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/build/pytest-3.10.1/.hypothesis/examples')
rootdir: /build/pytest-3.10.1, inifile: tox.ini
plugins: hypothesis-4.7.3
collected 2305 items / 1 deselected

testing/acceptance_test.py ............................................. [  1%]
.....x...................                                                [  3%]
testing/deprecated_test.py ......................                        [  3%]
testing/test_assertion.py ..........F

=================================== FAILURES ===================================
_ TestImportHookInstallation.test_installed_plugin_rewrite[development-plain] __

self = <test_assertion.TestImportHookInstallation object at 0x7ffff36d7dd8>
testdir = <Testdir local('/build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0')>
mode = 'plain', plugin_state = 'development'

    @pytest.mark.parametrize("mode", ["plain", "rewrite"])
    @pytest.mark.parametrize("plugin_state", ["development", "installed"])
    def test_installed_plugin_rewrite(self, testdir, mode, plugin_state):
        # Make sure the hook is installed early enough so that plugins
        # installed via setuptools are rewritten.
        testdir.tmpdir.join("hampkg").ensure(dir=1)
        contents = {
            "hampkg/__init__.py": """
                import pytest
    
                @pytest.fixture
                def check_first2():
                    def check(values, value):
                        assert values.pop(0) == value
                    return check
            """,
            "spamplugin.py": """
            import pytest
            from hampkg import check_first2
    
            @pytest.fixture
            def check_first():
                def check(values, value):
                    assert values.pop(0) == value
                return check
            """,
            "mainwrapper.py": """
            import pytest, pkg_resources
    
            plugin_state = "{plugin_state}"
    
            class DummyDistInfo(object):
                project_name = 'spam'
                version = '1.0'
    
                def _get_metadata(self, name):
                    # 'RECORD' meta-data only available in installed plugins
                    if name == 'RECORD' and plugin_state == "installed":
                        return ['spamplugin.py,sha256=abc,123',
                                'hampkg/__init__.py,sha256=abc,123']
                    # 'SOURCES.txt' meta-data only available for plugins in development mode
                    elif name == 'SOURCES.txt' and plugin_state == "development":
                        return ['spamplugin.py',
                                'hampkg/__init__.py']
                    return []
    
            class DummyEntryPoint(object):
                name = 'spam'
                module_name = 'spam.py'
                attrs = ()
                extras = None
                dist = DummyDistInfo()
    
                def load(self, require=True, *args, **kwargs):
                    import spamplugin
                    return spamplugin
    
            def iter_entry_points(name):
                yield DummyEntryPoint()
    
            pkg_resources.iter_entry_points = iter_entry_points
            pytest.main()
            """.format(
                plugin_state=plugin_state
            ),
            "test_foo.py": """
            def test(check_first):
                check_first([10, 30], 30)
    
            def test2(check_first2):
                check_first([10, 30], 30)
            """,
        }
        testdir.makepyfile(**contents)
        result = testdir.run(
            sys.executable, "mainwrapper.py", "-s", "--assert=%s" % mode
        )
        if mode == "plain":
            expected = "E       AssertionError"
        elif mode == "rewrite":
            expected = "*assert 10 == 30*"
        else:
            assert 0
>       result.stdout.fnmatch_lines([expected])
E       Failed: nomatch: 'E       AssertionError'
E           and: '============================= test session starts =============================='
E           and: 'platform linux -- Python 3.7.3, pytest-3.10.1, py-1.7.0, pluggy-0.12.0'
E           and: "hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/.hypothesis/examples')"
E           and: 'rootdir: /build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0, inifile:'
E           and: 'plugins: hypothesis-4.7.3'
E           and: 'collected 2 items'
E           and: ''
E           and: 'test_foo.py EE'
E           and: ''
E           and: '==================================== ERRORS ===================================='
E           and: '____________________________ ERROR at setup of test ____________________________'
E           and: 'file /build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/test_foo.py, line 1'
E           and: '  def test(check_first):'
E           and: "E       fixture 'check_first' not found"
E           and: '>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_xml_attribute, record_xml_property, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory'
E           and: ">       use 'pytest --fixtures [testpath]' for help on them."
E           and: ''
E           and: '/build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/test_foo.py:1'
E           and: '___________________________ ERROR at setup of test2 ____________________________'
E           and: 'file /build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/test_foo.py, line 4'
E           and: '  def test2(check_first2):'
E           and: "E       fixture 'check_first2' not found"
E           and: '>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_xml_attribute, record_xml_property, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory'
E           and: ">       use 'pytest --fixtures [testpath]' for help on them."
E           and: ''
E           and: '/build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/test_foo.py:4'
E           and: '=========================== 2 error in 0.02 seconds ============================'
E       remains unmatched: 'E       AssertionError'

/build/pytest-3.10.1/testing/test_assertion.py:237: Failed
----------------------------- Captured stdout call -----------------------------
running: /nix/store/0f331h72l3v42szvcqyc9pqg79ghid9m-python3-3.7.3/bin/python3.7 mainwrapper.py -s --assert=plain
     in: /build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0
============================= test session starts ==============================
platform linux -- Python 3.7.3, pytest-3.10.1, py-1.7.0, pluggy-0.12.0
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/.hypothesis/examples')
rootdir: /build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0, inifile:
plugins: hypothesis-4.7.3
collected 2 items

test_foo.py EE

==================================== ERRORS ====================================
____________________________ ERROR at setup of test ____________________________
file /build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/test_foo.py, line 1
  def test(check_first):
E       fixture 'check_first' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_xml_attribute, record_xml_property, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/test_foo.py:1
___________________________ ERROR at setup of test2 ____________________________
file /build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/test_foo.py, line 4
  def test2(check_first2):
E       fixture 'check_first2' not found
>       available fixtures: cache, capfd, capfdbinary, caplog, capsys, capsysbinary, doctest_namespace, monkeypatch, pytestconfig, record_property, record_xml_attribute, record_xml_property, recwarn, tmp_path, tmp_path_factory, tmpdir, tmpdir_factory
>       use 'pytest --fixtures [testpath]' for help on them.

/build/pytest-of-nixbld/pytest-0/test_installed_plugin_rewrite0/test_foo.py:4
=========================== 2 error in 0.02 seconds ============================
=========================== short test summary info ============================
FAIL testing/test_assertion.py::TestImportHookInstallation::()::test_installed_plugin_rewrite[development-plain]
XFAIL testing/acceptance_test.py::TestInvocationVariants::()::test_noclass_discovery_if_not_testcase
  decide: feature or bug
======== 1 failed, 101 passed, 1 deselected, 1 xfailed in 28.96 seconds ========
```

</details>

### Steps to reproduce

`nix-build '<nixpkgs>' -A beets`

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 5.0.18, NixOS, 19.09pre180487.d35692126f3 (Loris)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2.2`
 - channels(root): `"nixos-19.09pre184734.721b391f930"`
 - channels(nagisa): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

--- a/pkgs/development/python-modules/soco/default.nix
+++ b/pkgs/development/python-modules/soco/default.nix
@@ -1,22 +1,28 @@
 { lib, buildPythonPackage, fetchPypi, xmltodict, requests
 
 # Test dependencies
-, pytest_3, pytestcov, coveralls, pylint, flake8, graphviz, mock, sphinx
+, pytest, pytestcov, coveralls, pylint, flake8, graphviz, mock, sphinx
 , sphinx_rtd_theme
 }:
 
 buildPythonPackage rec {
   pname = "soco";
-  version = "0.16";
+  version = "0.17";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "7bed4475e3f134283af1f520a9b2e6ce2a8e69bdc1b58ee68528b3d093972424";
+    sha256 = "15zw6i5z5p8vsa3lp20rjizhv4lzz935r73im0xm6zsl71bsgvj8";
   };
+
+  postPatch = ''
+    # https://github.com/SoCo/SoCo/pull/670
+    substituteInPlace requirements-dev.txt \
+      --replace "pytest-cov>=2.4.0,<2.6" "pytest-cov>=2.4.0"
+  '';
 
   propagatedBuildInputs = [ xmltodict requests ];
   checkInputs = [
-    pytest_3 pytestcov coveralls pylint flake8 graphviz mock sphinx
+    pytest pytestcov coveralls pylint flake8 graphviz mock sphinx
     sphinx_rtd_theme
   ];
 

